frontend/eda/event-streams/hooks/useSelectEventStreams.tsx

```diff
@@ -0,0 +1,55 @@
+import { useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+import { edaAPI } from '../../../eda/common/eda-utils';
+import { useEventStreamFilters } from './useEventStreamFilters';
+import { useEventStreamColumns } from './useEventStreamColumns';
+import { EdaEventStream } from '../../interfaces/EdaEventStream';
+import { useEdaView } from '../../common/useEventDrivenView';
+import { MultiSelectDialog, usePageDialog } from '../../../../framework';
+
+export function useSelectEventStreams(eventStreamType?: number, title?: string) {
```

Comment:
If we remove `PageFormEventStreamSelect`, we may be able to remove `useSelectEventStreams` and `SelectEdaEventStreams` too